docs/src/release_notes/21.8.0.rst

```diff
@@ -0,0 +1,101 @@
+21.8.0
+======
+
+Direct runner on GKE
+--------------------
+
+Klio can now be run on the additional runner of ```DirectOnGKE``.
+
+Under the hood, when using the new DirectGKERunner with a Klio job, Klio creates a GKE deployment. When GKE starts the containers for the deployment, it runs the Klio job in DirectRunner mode (just like you would locally with ``klio job run --direct-runner``.
+
+The difference between DirectGKERunner and DirectRunner is that the latter automatically acknowledges messages it reads from the Pub/Sub queue before running it through the defined pipeline. This logic has been adapted on DirectGKERunner to acknowledge the message once it’s considered done running through the pipeline (i.e. either it successfully completed the pipeline, was filtered out, or was dropped due to an error in processing). Without this ability, if a container gets OOM-killed or the job gets otherwise interrupted, then the failing or in-progress messages do not return to the Pub/Sub queue for re-delivery. Those messages would be lost.
+
+Using the DirectGKERunner for a Klio job comes with inherent limitations:
+
+The Klio job must:
+    * be a streaming job (and therefore read event input from a Pub/Sub subscription or topic)
+    * not need any aggregation transforms (e.g. GroupByKey, CombinePerKey, Distinct, etc). Reshuffle & Flatten are also not supported.
+    * not process non-KlioMessages (klio-job.yaml::job_config.allow_non_klio_messages)
+    * be deployed manually (i.e. via ``klio job run [OPTIONS]``).
+    * not require GPUs
+
+* Observability
+
+Logs can still be viewed via kubectl logs.
+For metrics, support has been added for **shumway** [#f0]_ with which metrics can be viewed in Grafana.
```

Comment:
I believe Shumway technically adds support for `ffwd`, which may or may not be sending metrics to Grafana (also ffwd open-source is no longer maintained).

Reply:
Ah i see what you mean. Rephrasing this to be more general wording and leading grafana out of it and maybe for more specific spotify release notes we can get more into integrating with grafana.